### DISCUSSION TITLE
RSS feed for remix.run/blog

### DISCUSSION THREAD

**Discussion by tordans** [remix-run/remix#7506] [Proposals]
I would like to subscribe to updates on https://remix.run/blog. However, it looks like there is no RSS feed available.

Would it be possible to add one? 

I also could not find the source code for this part of the side, so I was not able to verify if there is a feed but I just did not find it.

**Comment by sergiodxa** [comment]
> I also could not find the source code for this part of the side, so I was not able to verify if there is a feed but I just did not find it.

The repository of the remix.run website is not open, but I just checked it and there's no RSS feed, I agree it will be nice to have one.
